RELEASE/scripts/autoscend/iotms/mr2018.ash

```diff
@@ -285,7 +285,8 @@ boolean fantasyRealmToken()
 	{
 		return false;
 	}
-	handleFamiliar($familiar[none]);
+	set_property("auto_disableFamiliarChanging", true);
+	use_familiar($familiar[none]);
```

Comment:
needs a 100% familiar check. or it will break 100% familiar run to get do fantasy realm

Reply:
The line right above the section GitHub is showing you is `if(is100FamRun())`